specs/linux_file_system_hierarchy.md

```diff
@@ -229,17 +229,65 @@ and other compiler-specific documentation.
 
 The corresponding library or libraries shall be placed in `$libdir`, see below.
 
-### `/usr/lib/`
+### Libraries, system programs, and program assets
+
+This section describes the directories used to store shared libraries,
+internal binaries or other programs that are not regularly invoked from a shell
+(and thus should not be placed in the directories in `$PATH`),
+and other static files that are part of program installations.
+
+There are two main schemes for shared libraries.
+On "multiarch" systems,
```

Comment:
Could we go for something else, e.g.
```suggestion
On *multiarch* systems,
```
I think that stands out a bit easier, doesn't have an additional glyph when grepping in the rendered text and would translate more nicely to other output formats.

Same below.